tests/test.robot

```diff
@@ -0,0 +1,77 @@
+*** Settings ***
```

Comment:
Is this test case meant for alcohol project?

Reply:
No sir this is for Hariprasath project Smart-Blind project sir